src/io/flutter/preview/PreviewView.java

```diff
@@ -264,6 +279,42 @@ Icon getIcon() {
     return icon;
   }
 
+  /**
+   * Return the string that is suitable for speed search. It has every name part separted so that we search only inside individual name
+   * parts, but not in their accidential concatenation.
+   */
+  @NotNull
+  String getSpeedSearchString() {
```

Comment:
private?

Reply:
In Java private is stronger than in Dart, it prevents access even for classes in the same file. So, default visibility.